### ISSUE TITLE
Hangs at scraping

### ISSUE THREAD

**Issue by ToughGuyKunio** [quinton-ashley/nostlan#16]
Nostlan hangs at scraping

Version: 1.7.7
OS: Manjaro Linux

Error:
```
Uncaught (in promise) TypeError: Cannot read property 'localeCompare' of undefined
    at /tmp/.mount_nostla4uVQ4V/resources/app.asar/scrape/scraper.js:194
    at Array.sort (<anonymous>)
    at Scraper.loadImages (/tmp/.mount_nostla4uVQ4V/resources/app.asar/scrape/scraper.js:194)
    at async viewerLoad (/tmp/.mount_nostla4uVQ4V/resources/app.asar/views/js/index.js:902)
    at async reload (/tmp/.mount_nostla4uVQ4V/resources/app.asar/views/js/index.js:188)
    at async CUI.cui.onAction (/tmp/.mount_nostla4uVQ4V/resources/app.asar/views/js/index.js:710)
    at async CUI.doAction (/tmp/.mount_nostla4uVQ4V/resources/app.asar/node_modules/contro-ui/dist/js/contro-ui.js:224)
    at async CUI.buttonPressed (/tmp/.mount_nostla4uVQ4V/resources/app.asar/node_modules/contro-ui/dist/js/contro-ui.js:540)
```

It seems to hang on different games each time I launch it. I also tried version 1.60 but just got a blank screen and the buttons in each corner, no menus where working.

**Comment by quinton-ashley** [comment]
I will have a fix for this in the next version sorry!

**Comment by quinton-ashley** [comment]
Let me know if the 1.8 beta version fixes that problem!

**Comment by ToughGuyKunio** [comment]
I'm not really sure how to compile it, is there a documentation? Otherwise can you make an AppImage?

**Comment by quinton-ashley** [comment]
Whoops sorry, I thought I had already done it.

**Comment by quinton-ashley** [comment]
I uploaded the AppImage just now!

**Comment by quinton-ashley** [comment]
I released v1.8.0 today.  I think it will fix that problem but let me know!

**Comment by quinton-ashley** [comment]
Can confirm that this was fixed.
